src/go/rpk/pkg/cli/redpanda/admin/config/config.go

```diff
@@ -70,10 +73,71 @@ func newLogLevelCommand(fs afero.Fs, p *config.Params) *cobra.Command {
 	}
 	cmd.AddCommand(
 		newLogLevelSetCommand(fs, p),
+		newLogLevelGetCommand(fs, p),
 	)
 	return cmd
 }
 
+func newLogLevelGetCommand(fs afero.Fs, p *config.Params) *cobra.Command {
+	var host string
+
+	cmd := &cobra.Command{
+		Use:   "get [LOGGERS...]",
+		Short: "Get broker logger's log level",
+		Long: `Get broker logger's log level.
+
+This command obtains the broker logger's log level. Each Redpanda
+broker has many loggers, and each can be individually returned.
+
+You can specify many loggers at once by separating each logger name with a space.
+To return all loggers, you can leave the logger argument empty or pass "all". 
+To see all possible loggers, run 'redpanda --help-loggers' in your node.
```

Comment:
```suggestion
To list all available loggers, run 'redpanda --help-loggers' in your broker.
```
Pardon my ignorance -- would this happen to be an rpk command as well (i.e. `rpk redpanda --help-loggers`)?

Reply:
`redpanda --help-loggers` is a classic way. I kinda feel this message is a duplicate. In the above, you said:

```
To return all loggers, you can leave the logger argument empty or pass "all". 
```